Bluetooth as localization technology (#18)
ICPS-MicroCity/amusement-park-analysis: doc/sections/2-technologies.tex

%     f^{*} = \arg \max_{f_i \in F} \ \eta(f, f_i)
% \end{equation}

% $L(f^{*})$ is the corresponding location of $f^{*}$ which represent the estimated place.

\subsection{Bluetooth as outdoor localization system}
Bluetooth is a data transmission standard for personal wireless networks. It provides a standard way to exchange information between
devices through a short-range frequency capable of detecting devices covered by the radio signal within about ten meters by putting them
in communication with each other.

Despite being a technology conceived more than 20 years ago, it boasts massive adoption in many contexts such as medical, industrial, and in
recent years in the Internet of Things (IoT). This standard was designed with the goal of achieving low power consumption, short range
and a low cost of production. Over the years, there have been several updates to the protocol, aimed at improving on the one hand the efficiency of

> communication by enabling higher transmission frequencies and on the other hand to improve the energy efficiency of devices.

Reviewer: ```suggestion
communication by enabling higher transmission frequencies and on the other hand improving the energy efficiency of devices.
```

Bluetooth was not designed to perform localization functionality. However, it is possible to exploit certain
features of the protocol to make a more or less precise estimate of a device's location. In particular, the technique most

> used to achieve this is that based on the signal strength identified by the RSSI (Receives Signal Strength Indicator).

Reviewer: ```suggestion
used to achieve this is based on the signal strength identified by the RSSI (Receives Signal Strength Indicator).
```

> Other techniques, such as fingerprint-base localization can be employed to estimate position.

Reviewer: ```suggestion
Other techniques, such as fingerprint-based localization can be employed to estimate position.
```
